oonidata/observations.py

```diff
@@ -214,19 +214,19 @@ def from_measurement(
         # hrro.response_headers_list = http_transaction.response.headers_list_bytes
 
         hrro.response_header_location = get_first_http_header(
-            "location", http_transaction.response.headers_list_bytes
+            "location", http_transaction.response.headers_list_bytes or []
         )
         hrro.response_header_server = get_first_http_header(
-            "server", http_transaction.response.headers_list_bytes
+            "server", http_transaction.response.headers_list_bytes or []
         )
 
         try:
-            prev_request = requests_list[idx + 1]
+            prev_request = requests_list[idx + 1] # type: ignore
```

Comment:
I don't immediately see why you're ignoring types here. Would you mind adding a comment to the codebase that explains why here and in a bunch of subsequent locations you're forced to ignore types?